foxpuppet/api/windows.py

```diff
@@ -0,0 +1,173 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
I'm against moving everything into an `api` directory. FoxPuppet is entirely an API, so I feel the extra level is redundant. It also makes reviewing very difficult when combined with other changes.